src/mlpack/methods/ann/ffn.hpp

```diff
@@ -142,6 +142,16 @@ class FFN
   void Predict(arma::mat& predictors, arma::mat& results);
 
   /**
+   * Predict the responses to a given set of predictors. The responses will
+   * reflect the output of the given output layer as returned by the
+   * output layer function.
+   *
+   * @param predictors Input predictors.
+   * @return Output predictions of responses into.
+   */
+  arma::mat Predict(arma::mat predictors);
+
```

Comment:
I still don't like the idea to return the output, basically because it's the only place we would do this if I switch to another method I might wonder why I can't call ``output = model.Predict(input)``.